pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -205,9 +210,16 @@ const (
 
 	// ClusterConditionNone is a special value indicating that no cluster condition should be set
 	ClusterConditionNone ClusterConditionType = ""
-
-	ReasonClusterUpdateSuccessful = "ClusterUpdateSuccessful"
-	ReasonClusterUpdateInProgress = "ClusterUpdateInProgress"
+	// This condition is met when a CSI migration is ongoing and the CSI
+	// migration feature gates are activated on the Kubelets of all the nodes.
+	// When this condition is `true` CSIMigration{provider}Complete can be
+	// enabled.
+	ClusterConditionCSIKubeletMigrationCompleted ClusterConditionType = "CSIKubeletMigrationCompleted"
+
+	ReasonClusterUpdateSuccessful             = "ClusterUpdateSuccessful"
+	ReasonClusterUpdateInProgress             = "ClusterUpdateInProgress"
+	ReasonClusterCSIKubeletMigrationCompleted = "CSIKubeletMigrationSuccess"
+	ReasonClusterCCMMigrationInProgress       = "CSIKubeletMigrationInProgress"
```

Comment:
Those 3 reasons are not used at all.

Reply:
I introduced the last 2, and it's true they are not used yet. They will be used by: https://github.com/kubermatic/kubermatic/issues/6551

I don't have problems either in keeping or removing them.